### PR TITLE
feat: add telemetry

### PR DESCRIPTION
**Pull request by converge**
Adding telemetry to send new health metrics to track cluster availability

Signed-off-by: João Vanzuita <joao@kubeshop.io>

### CODE REVIEW

**Review by johndietz — APPROVED** [review]

File: pkg/constants.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package pkg
+
+const SegmentIOWriteKey = "0gAYkX5RV3vt7s4pqCOOsDb6WHPLT30M"
+const MetricHealth = "kubefirst.gh.mgmt.kubefirst.com.healthy"
```
Comment (converge) [inline pkg/constants.go]: I would just keep it "healthz" to follow k8s/google pattern

**Review by pagottoo — APPROVED** [review]
lgtm

File: pkg/constants.go (hunk 1 above)
Comment (pagottoo) [reply pkg/constants.go]: Could these values be env vars? Why are they hard-coded?

Comment (pagottoo) [reply pkg/constants.go]: Could these values be env vars? Why are they hard-coded?

**Pull request merged**